quilt/tools/store.py

```diff
@@ -141,14 +149,33 @@ def ls_packages(self):
         """
         List packages in this store.
         """
```

Comment:
time extract this block into a reusable method/function ?

Reply:
The implementation could certainly be cleaned up. As of now, it's only used by quilt ls. Can you suggest a name & signature for a better reusable function? Perhaps one that returned a list or iterator of Package objects?